### PR TITLE
Wayland: Added support for output-power-management-v1

### PR DESCRIPTION
**Pull request by anriha**
This adds basic support for output-power-management protocol. It is dependant on flacjacket/pywlroots#73

### CODE REVIEW

File: libqtile/backend/wayland/core.py [hunk 1]
```diff
@@ -155,6 +156,8 @@ def __init__(self):
         XdgOutputManagerV1(self.display, self.output_layout)
         ScreencopyManagerV1(self.display)
         GammaControlManagerV1(self.display)
+        self._output_power_manager = OutputPowerManagerV1(self.display)
+        self.add_listener(self._output_power_manager.set_mode_event, self._on_output_power_manager_set_mode)
```
Comment (m-col) [inline libqtile/backend/wayland/core.py:160]: It looks like we don't need to keep around a handle to the manager as an attribute here.
```suggestion
        output_power_manager = OutputPowerManagerV1(self.display)
        self.add_listener(output_power_manager.set_mode_event, self._on_output_power_manager_set_mode)
```

**Review by m-col — COMMENTED** [review]
Just one comment but overall this looks great, thank you very much!

One or two of the lines look quite long, so the CI might complain. You can just run `black` on the code and it'll fix it up.

Please could you also add a line to the changelog?

**Comment by m-col** [comment]
It looks like black has reformatted a couple of unrelated lines, which I suspect is because it didn't find the `pyproject.toml`. Could you re-run it - you might need to be in the root folder.

**Comment by anriha** [comment]
Now it should be OK, is the line in CHANGELOG ok like this? https://github.com/anriha/qtile/blob/c8280a30a757ad74297ca692aecd15434edd53d2/CHANGELOG#L4

**Comment by m-col** [comment]
Yep it all looks perfect, thanks! I'll wait until the PR on pywlroots is merged to start the CI otherwise it will just fail. Then if they are all good then we can merge!

**Comment by anriha** [comment]
flacjacket/pywlroots#73 was merged this should now work

**Comment by m-col** [comment]
Looks like the docs aren't building. I think you need to add `wlroots.wlr_types.output_power_management_v1` to `MOCK_MODULES` in `docs/conf.py`

**Comment by m-col** [comment]
Perfect. It'll merge if/when the tests pass. Thanks again!

**Pull request merged**